version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "2.5.0"
+version in ThisBuild := "2.6.0"
```

Comment:
Typically this is bumped automatically by the release plugin and should not be touched in a PR.